### ISSUE TITLE
public API detection: problem with multi-line enumerator definitions and comment at the end of last line

### ISSUE THREAD

**Issue by guwirth** [SonarOpenCommunity/sonar-cxx#2161]
* using SQ 7.9 LTS with cxx plugin 1.3.3
* documentation for the enumerator B is not detected by the public API visitor:
```C++
typedef enum {
    A = 0, ///< docu A
    B =  
        1, ///< docu B
} MyEnum;
```
